examples/plot_employee_salaries.py

```diff
@@ -0,0 +1,106 @@
+"""
+Predicting the salary of employees
+==================================
+
+Now that we understand how the similarity encoder works, let's 
```

Comment:
You cannot start an example like this: users don't read exemples in a linear way